packages/main/src/resolvers/conversation.ts

```diff
@@ -180,6 +195,19 @@ export const ConversationMutations: ConversationMutationsResolvers = {
     return C.mustGetConversation(id)
   },
 
+  async saveDraft(_parent, { accountId, id, content }) {
+    const account = mustGetAccount(accountId)
+    const conversation = C.mustGetConversation(id)
+    schedule(
+      actions.saveDraft({
+        box: { attribute: "\\Drafts" },
+        accountId,
+        message: composeReply({ account, content, conversation })
+      })
```

Comment:
This is going to create a lot of stored messages. Can we replace the existing draft if there is one?